skimage/segmentation/tests/test_active_contour_model.py

```diff
@@ -135,14 +135,14 @@ def test_coord_raises():
     init = np.array([x, y]).T
     # coordinates='xy' is not valid
     with pytest.raises(ValueError):
-        snake = active_contour(gaussian(img, 3), init,
-                               boundary_condition='periodic', alpha=0.015,
-                               beta=10, w_line=0, w_edge=1, gamma=0.001,
-                               max_num_iter=100, coordinates='xy')
+        active_contour(gaussian(img, 3), init,
+                        boundary_condition='periodic', alpha=0.015,
+                        beta=10, w_line=0, w_edge=1, gamma=0.001,
+                        max_num_iter=100, coordinates='xy')
 
     # coordinates=None is not valid
     with pytest.raises(ValueError):
-        snake = active_contour(gaussian(img, 3), init,
-                               boundary_condition='periodic', alpha=0.015,
-                               beta=10, w_line=0, w_edge=1, gamma=0.001,
-                               max_num_iter=100, coordinates=None)
+        active_contour(gaussian(img, 3), init,
+                        boundary_condition='periodic', alpha=0.015,
+                        beta=10, w_line=0, w_edge=1, gamma=0.001,
+                        max_num_iter=100, coordinates=None)
```

Comment:
```suggestion
        active_contour(gaussian(img, 3), init,
                       boundary_condition='periodic', alpha=0.015,
                       beta=10, w_line=0, w_edge=1, gamma=0.001,
                       max_num_iter=100, coordinates=None)
```
indentation was off by 1?